.github/workflows/integration.yml

```diff
@@ -114,6 +114,8 @@ jobs:
                         "systemd-networkd",
                 ]
                 test: [
+                        "20",
```

Comment:
Test 20 has some other issues on Arch that needs similar changes to https://github.com/dracut-ng/dracut-ng/pull/381

You should either not add tests-20 yet or try to fix it please.

```
--- a/test/TEST-20-NFS/test.sh
+++ b/test/TEST-20-NFS/test.sh
@@ -366,7 +366,7 @@ test_setup() {
     # We do it this way so that we do not risk trashing the host mdraid
     # devices, volume groups, encrypted partitions, etc.
     "$DRACUT" -l -i "$TESTDIR"/server/overlay / \
-        -m "bash rootfs-block kernel-modules qemu" \
+        -a "bash rootfs-block kernel-modules qemu" \
         -d "piix ide-gd_mod ata_piix ext4 sd_mod" \
         --nomdadmconf \
         --no-hostonly-cmdline -N \
@@ -416,7 +416,7 @@ test_setup() {
     )
     # Make server's dracut image
     "$DRACUT" -l -i "$TESTDIR"/overlay / \
-        -m "bash rootfs-block debug kernel-modules watchdog qemu network-legacy" \
+        -a "bash rootfs-block debug kernel-modules watchdog qemu network-legacy" \
         -d "af_packet piix ide-gd_mod ata_piix ext4 sd_mod e1000 i6300esb" \
         --no-hostonly-cmdline -N \
         -f "$TESTDIR"/initramfs.server "$KVERSION" || return 1
```

Reply:
Yeah, that's for a different PR to address.
I'm tempted to add a section for gentoo + networkd, since that's where I have been testing most locally.